src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -651,4 +683,15 @@ private DslPart createResponseForGetTaskForTaskType() {
                         .datetime("next_hearing_date", "yyyy-MM-dd'T'HH:mm:ssZ")
                 )).build();
     }
+
+    private DslPart createResponseForGetTaskWithCompletionProcess() {
+        return newJsonBody(
+            o -> o
+                .minArrayLike("tasks", 1, 1, task -> {
+                    createResponseForGetTask();
+                    task.stringMatcher("termination_process",
+                                       "EXUI_USER_COMPLETION|EXUI_CASE-EVENT_COMPLETION",
+                                       "EXUI_USER_COMPLETION");
```

Comment:
Not sure if this represents that the API contract is such that the `termination_process` can be not present?  It's not a mandatory field and it could be `null` or empty in the DB e.g. historic records and cancelled Tasks.  I guess in this case we always return the field and value but that's not the actual contract?

Reply:
After discussion we have another test which covers the scenario where this field is absent.  This should be sufficient so resolving this now.